utils/create.c

```diff
@@ -128,6 +128,8 @@ static char* tr_getcwd(void)
 
 int tr_main(int argc, char* argv[])
 {
+    char buf[128];
+    uint32_t current, last;
```

Comment:
As we're using C99, please move declarations down closer to their first use, combining with initialization when possible. Also, please prefer declaring each variable separately (on its own line in its own statement), even when the type is the same.